src/spec-lang/ast/declarations/annotation.ts

```diff
@@ -0,0 +1,16 @@
+import { Range, SNode } from "../node";
+export enum AnnotationType {
+    IfSucceeds = "if_succeeds",
+    Invariant = "invariant",
+    Define = "define"
```

Comment:
A side note (not related to the current changes): was always wondering, why do we not have `IfAborts` here, but have it in detection regex?

Reply:
Just being silly. Will remove it from the detection regex.

Reply:
Fixed in 031fe1c.